Dlls/UniRx.SystemReactive/UniRx.SystemReactive.csproj

```diff
@@ -9,14 +9,14 @@
     <DefineConstants>$(DefineConstants);UniRxLibrary SystemReactive</DefineConstants>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageId>UniRx.SystemReactive</PackageId>
-    <Version>1.0.5</Version>
+    <Version>1.1.0</Version>
     <Authors>OrangeCube</Authors>
     <Company>OrangeCube</Company>
     <Product>UniRx.SystemReactive</Product>
     <Description>UniRx in System.Reactive namespace</Description>
     <Copyright>OrangeCube</Copyright>
     <PackageTags>UniRx, Rx, reactive extensions</PackageTags>
-    <PackageReleaseNotes>[fixed] IProgress is missing in net45 and netstandard.</PackageReleaseNotes>
+    <PackageReleaseNotes>TypeForward is suspend.</PackageReleaseNotes>

```

Comment:
Abandon using TypeForwarding. We decided to use the UniRx implementation even in net46 or later.